boards/arm/mimxrt595_evk/board.c

```diff
@@ -6,19 +6,45 @@
 #include <zephyr/init.h>
 #include "fsl_power.h"
 #include <zephyr/pm/policy.h>
+#include "board.h"
+
+#ifdef CONFIG_FLASH_MCUX_FLEXSPI_XIP
+	#include "flash_clock_setup.h"
```

Comment:
No need for spaces at the begininng

Reply:
Thanks @mmahadevan108 , I removed that indent